### DISCUSSION TITLE
Installation issue after upgrade from 4 to 5

### DISCUSSION THREAD

**Discussion by cbrownrome** [ampache/ampache#3540] [Q&A] [answered]
I've got this error after pass 3 of installation wizard
`La selezione del database è fallita: ["42S02",1146,"Table 'ampache.session' doesn't exist"]
`
on linux mint 19

**Comment by mitchray** [comment, marked as answer]
This solution might work for you: https://github.com/ampache/ampache/issues/3349#issuecomment-1326983690

**Reply by cbrownrome** [reply]
That's ok for ampache.session, now it remains to solve PHP intl extension 	ERROR in next debug page. Tried:
`sudo apt-get install php7.4-intl` and obtain `E: Impossibile trovare il pacchetto php7.4-intl` also enabled php-intl in php.ini removing semicolumn

**Reply by lachlan-00** [reply]
mint might just simply be `php-intl`

https://community.linuxmint.com/software/view/php-intl

gettext is required too if missing
https://github.com/ampache/ampache/wiki/Ampache-next-Changes#php-74-php-intl-and-php-gettext-are-required

**Reply by cbrownrome** [reply]
Trying a new install of php-intl, I discovered a php8.2-intl installed, so I removed it. Then enabled php-intl and php-gettext both in php.ini (both installed). Again ERROR after web server restart and new config creation.
Going haed in tutorial I obtain:
```
$ composer install --prefer-source --no-interaction
Loading composer repositories with package information
Warning from https://packagist.org: Support for Composer 1 is deprecated and some packages will not be available. You should upgrade to Composer 2. See https://blog.packagist.com/deprecating-composer-1-support/
Info from https://packagist.org: #StandWithUkraine
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested PHP extension ext-intl * is missing from your system. Install or enable PHP's intl extension.
```

when I run composer
(also Is peraphs ext-intl a different extension from php-intl?)
